### PR TITLE
chore: Bootstrap AzureDevOps plugin

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Related to https://github.com/cloudquery/cloudquery/issues/5511. This bootstraps the AzureDevOps plugin.
It doesn't implement anything yet, since I'm splitting it into multiple PRs to make it easier to review, hence it's still a `chore`.
This PR includes most of the steps in https://github.com/cloudquery/cloudquery/blob/51acb48a3c74c2f478de260bf20d015dc38acee7/contributing/adding_a_new_plugin_to_cq_monorepo.md

### CODE REVIEW

File: plugins/source/azuredevops/main.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/azuredevops/resources/plugin"
+	"github.com/cloudquery/plugin-sdk/serve"
+)
+
+const sentryDSN = "https://81e8135dfd0e42629810a434e0dd72cb@o1396617.ingest.sentry.io/4504317272915968"
```
Comment (erezrokah) [inline plugins/source/azuredevops/main.go:8]: This DSN is correct (created a new project)

**Comment by yevgenypats** [comment]
Looks good. Quick Q: Wasn't Azure Devops migrated to GitHub actions (maybe not officially yet but defacto or still people/companies use it?)?

**Comment by erezrokah** [comment]
> Wasn't Azure Devops migrated to GitHub actions (maybe not officially yet but defacto or still people/companies use it?)?

From a quick Google search it's not deprecated. Also even if it is, I think it will be useful to know what to migrate to GitHub using CloudQuery :)

**Comment by erezrokah** [comment]
CI should be fixed by https://github.com/cloudquery/cloudquery/pull/5563

**Pull request merged**